app/uk/gov/hmrc/testuser/config/microserviceGlobal.scala

```diff
@@ -62,3 +64,42 @@ object MicroserviceGlobal extends DefaultMicroserviceGlobal with RunMode with Mi
 
   override val authFilter = Some(MicroserviceAuthFilter)
 }
+
+case class RegistrationRequest(serviceName: String, serviceUrl: String, metadata: Option[Map[String, String]] = None)
+
+object ServiceLocatorConnector extends ServicesConfig {
```

Comment:
ServiceLocatorConnector is already in the library play-hmrc-api.
https://github.com/hmrc/play-hmrc-api/blob/master/src/main/scala/uk/gov/hmrc/api/connector/ServiceLocatorConnector.scala